src/explore-education-statistics-admin/src/pages/release/edit-release/manage-datablocks/components/DataBlockDetailsForm.tsx

```diff
@@ -146,6 +148,11 @@ const DataBlockDetailsForm = ({
                 id="data-block-title"
                 name="title"
                 label="Table title"
+                onChange={e => {
+                  const newTitle = e.target.value;
+                  form.setValues({ ...form.values, title: newTitle });
```

Comment:
Shouldn't need to do this, think there's actually an error in the `FormFieldTextArea` change handler